blockchain/storage/keyvalue.go

```diff
@@ -284,6 +287,21 @@ func (s *KeyValueStorage) ReadTxLookup(hash types.Hash) (types.Hash, bool) {
 	return types.BytesToHash(blockHash), true
 }
 
+// ReadLastProcessedEvent reads the last processed event info
+func (s *KeyValueStorage) ReadLastProcessedEvent(contractAddr string) (string, bool) {
+	data, ok := s.get(LAST_PROCESSED_EVENT, []byte(contractAddr))
+	if !ok {
+		return "", false
+	}
+
+	return string(data), true
+}
+
+// WriteLastProcessedEvent writes the last processed event info
+func (s *KeyValueStorage) WriteLastProcessedEvent(data string, contractAddr string) error {
+	return s.set(LAST_PROCESSED_EVENT, []byte(contractAddr), []byte(data))
+}
```

Comment:
I don't see any meaningful reason why this domain appears under blockchain/storage package. I thought you didn't want to use KeyValueStorage without modification. Why don't you use own storage?

Reply:
I don't see the reason to define and carry around a new DB instance, since we already have one present in Edge.
This is detailed in the RFC - we want to go with a single DB instance instead of managing a separate DB for a single value

Reply:
@Kourin1996 
Completely disregard this, we will use a separate DB instance that will be added in a separate configuration.

It will be a Key Value store, so we'll leave this out